styles/codesample.css

```diff
@@ -0,0 +1,15 @@
+
+/** Code snippet on module pages */
+
+.control {
+    cursor: pointer;
+}
+
+pre.codesample {
+    text-align: left;
+    background: transparent;
+}
+
+button pre.codesample, button pre code[class*="language-"] {
+    color: #6E6E6E;
+}
```

Comment:
Not sure if the code snippet could be styled here, or if that would be more complicated, but overall the new code snippet has worse contrast (gray on gray), which isn't great for accessibility reasons.